{{cookiecutter.project_name}}/steps/predict.py

```diff
@@ -11,7 +13,13 @@ def predict_batch(spark_session, model_uri, input_table_name, output_table_name)
     predict = mlflow.pyfunc.spark_udf(
         spark_session, model_uri, result_type="string", env_manager="conda"
     )
-    output_df = table.withColumn("prediction", predict(struct(*table.columns)))
+    output_df = (
+        table.withColumn("prediction", predict(struct(*table.columns)))
+        .withColumn("model_version", lit(model_version))
+        .withColumn("ts", lit(ts))
```

Comment:
Is there a reason we're adding a column for both the literal value and the casted value? Do we expect the cast to fail? Or since we're using the same column key we expect the latter cast calls to overwrite?

Reply:
`model_version` cast is necessary and I'll remove it. string works.

I'm not familiar enough with spark. Let me try `to_timestamp(lit(ts))` and see if it works.